lib/services/models.ts

```diff
@@ -178,10 +178,15 @@ export function getSequelizeTypeByDesignType(target: any, propertyName: string):
  */
 export function getModels(arg: Array<typeof Model | string>): Array<typeof Model> {
 
-  if (arg && typeof arg[0] === 'string') {
+  if (arg) {
 
     return arg.reduce((models: any[], dir) => {
 
+      if (typeof dir !== 'string') {
```

Comment:
@RobinBuschmann do we have tests on this method? (I don't think I saw any in `model.spec.ts`?) This would be good to have some tests. (array with Model and String elems / only string elems / only Model elems / empty array / null / object).

Reply:
@snewell92 No, we don't have any for this method only. @schmod  Can you add some tests as a part of this PR? This would also increase coverage :)